bundler/spec/commands/config_spec.rb

```diff
@@ -408,6 +408,14 @@
       expect(out).to eq "spec_run=true"
     end
 
+    it "list with credentials" do
+      bundle "config list", :env => { "BUNDLE_GEMS__MYSERVER__COM" => "user:password" }
+      expect(out).to eq "Settings are listed in order of priority. The top value will be used.\nspec_run\nSet via BUNDLE_SPEC_RUN: \"true\"\n\ngems.myserver.com\nSet via BUNDLE_GEMS__MYSERVER__COM: \"user:[REDACTED]\""
+
+      bundle "config list", :parseable => true, :env => { "BUNDLE_GEMS__MYSERVER__COM" => "user:password" }
+      expect(out).to eq "spec_run=true\ngems.myserver.com=user:password"
```

Comment:
As this spec proves, credentials are redacted from regular `bundle config` out, but not when using the `--parseable` flag, which seems like the right thing to do.

Reply:
Seems reasonable, but would be good to document this behaviour to prevent unexpected output.